agent_holder/src/credential/aggregate.rs

```diff
@@ -70,8 +70,6 @@ impl Aggregate for Credential {
     fn apply(&mut self, event: Self::Event) {
         use CredentialEvent::*;
 
-        info!("Applying event: {:?}", event);
```

Comment:
You sure we don't want to log when events are applied? You could also change the log level to `debug` instead.

Reply:
You're right, leaving them in as `debug` could still be beneficial :+1: